packages/editor-ui/src/components/Node.vue

```diff
@@ -126,6 +126,9 @@ export default mixins(
 		NodeIcon,
 	},
 	computed: {
+		isScheduledGroup (): boolean {
+			return this.nodeType?.group.includes('schedule') === true;
```

Comment:
just a nitpick: you can omit ` === true`, it returns a boolean anyways

Reply:
I can't without TS error because I'm using optional chaining so it would just return undefined, not boolean
<img width="1015" alt="CleanShot 2022-10-13 at 10 26 17@2x" src="https://user-images.githubusercontent.com/12657221/195543743-2740fc2e-4eb8-4aca-b220-364bf924eedb.png">

Reply:
Sorry, I see now, then just convert it to boolean with `!!`
```suggestion
			return !!this.nodeType?.group.includes('schedule');
```

Reply:
Isn't that pretty much identical? :D

Reply:
Yes, that's why I said nitpick. I always choose what's less character because I'm lazy to type ;)